config/auth.php

```diff
@@ -106,7 +106,7 @@
             'provider' => 'users',
             'email' => 'auth.emails.password',
             'table' => 'password_resets',
-            'expire' => 1440,
+            'expire' => 4320,
```

Comment:
Maybe worth throwing in a comment? (I'm just really bad at math 🙃 )
```suggestion
            // 72 hours.
            'expire' => 4320,
```

Reply:
oh good call @mendelB ! then we can see at a glance what this seemingly random number means!